### PR TITLE
Minor tweak + Wording change

### PR DESCRIPTION
**Pull request by ricetech**
- Make mobile hamburger menu icon & border less transparent (is currently somewhat faded)
- Change wording on homepage

### CODE REVIEW

File: src/app/core/navbar/navbar.component.css [hunk 1]
```diff
@@ -49,6 +49,12 @@
 
 .navbar-toggler {
   margin-right: 1em;
+  border-color: rgba(255, 255, 255, 0.2);
+}
+
+.navbar-toggler-icon {
+  /* Replace the hamburger icon image with a custom SVG that has custom colours */
+  background-image: url("data:image/svg+xml;charset=utf8,%3Csvg viewBox='0 0 32 32' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath stroke='rgba(255,255,255,0.7)' stroke-width='2' stroke-linecap='round' stroke-miterlimit='10' d='M4 8h24M4 16h24M4 24h24'/%3E%3C/svg%3E");
```
Comment (DabeerAzeez) [inline src/app/core/navbar/navbar.component.css]: Could this svg be saved into `/assets`? It seems to be cluttering up the CSS here.

File: src/app/pages/home/home.component.html [hunk 2]
```diff
@@ -68,7 +68,8 @@ <h2 class="mb-3 px-5 home-text header-text">What is a virtrolio?</h2>
         <div class="card mr-4 mb-5 rounded border-0">
           <h3 class="body-header-text">Why use Virtrolio?</h3>
           <p class="lead body-text">Using Virtrolio offers you an authentic yearbook signing experience, and
-            that means <em>privacy</em>. You can only view virtrolio messages sent directly to you (see our
+            that means <em>privacy</em>. Just like how only you can view your physical yearbook,
```
Comment (DabeerAzeez) [inline src/app/pages/home/home.component.html]:
```suggestion
            that means <em>privacy</em>. Just like how only you can view <em>your</em> physical yearbook,
```

File: src/app/pages/faq/faq.component.html [hunk 3]
```diff
@@ -259,6 +259,26 @@ <h2 class="mb-0">
...
+                  data-toggle="collapse" type="button">
+            Can I change my display name?
+          </button>
+        </h2>
+      </div>
+
+      <!-- Answer -->
+      <div aria-labelledby="s2q5h" class="collapse" data-parent="#faq-s2" id="s2q5c">
+        <div class="card-body card-align">
+          No, your display name is taken directly from Google. We suggest you use a personal Gmail account to sign up
```
Comment (ricetech) [inline src/app/pages/faq/faq.component.html]:
```suggestion
          No, your display name is taken directly from your Google Account. We suggest you use a personal Gmail account to sign up
```

File: src/app/pages/signing/signing.component.html [hunk 4]
```diff
@@ -169,3 +175,28 @@ <h5>Emojis</h5>
...
   </div>
 </div>
+
+<!-- Signing Experience  -->
+<div class="modal fade" id="signingExperience" tabindex="-1" role="dialog" aria-labelledby="info-title"
+     aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title" id="experience-title">I need a larger screen to use Virtrolio?</h4>
```
Comment (ricetech) [inline src/app/pages/signing/signing.component.html]:
```suggestion
        <h4 class="modal-title" id="experience-title">Do I need a larger screen to use Virtrolio?</h4>
```

File: src/app/pages/signing/signing.component.html [hunk 5]
```diff
@@ -169,3 +175,28 @@ <h5>Emojis</h5>
...
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        For most purposes, no. We designed the rest of Virtrolio to be used on a mobile OR desktop/laptop device, but we
+        suggest that you
+        try <em>signing</em> messages on a desktop or laptop computer instead of a phone or tablet. This is because
+        not only is it easier to write out a message when you have a larger screen, but some of the components (like
+        the colour pickers) are not as good-looking or easy to use on mobile because of what we have available.
```
Comment (ricetech) [inline src/app/pages/signing/signing.component.html]:
```suggestion
        the colour pickers) are not as good-looking or easy to use on mobile.
```

**Pull request merged**